cfme/tests/configure/test_default_views_cloud.py

```diff
@@ -26,54 +25,56 @@
 # TODO refactor for setup_provider parametrization with new 'latest' tag
 
 
-def set_and_test_default_view(group_name, view, page):
-    old_default = DefaultView.get_default_view(group_name, fieldset='Clouds')
-    DefaultView.set_default_view(group_name, view, fieldset='Clouds')
+def set_and_test_default_view(appliance, group_name, view, page):
```

Comment:
**Optional**

Consider calling the argument `expected_view` instead of `view` for clarity